lib/automation/report_runner.rb

```diff
@@ -57,7 +57,15 @@ def initialize(
         automation: nil
       )
         @sender = User.find_by(username: sender_username)
-        @receivers = User.where(username: receivers)
+        receivers_without_emails = receivers&.reject { |r| r.include? "@" }
+        if receivers_without_emails.present?
+          @group_receivers = Group.where(name: receivers_without_emails)
+          receivers_without_emails -= @group_receivers.pluck(:name)
+          @receivers = User.where(username: receivers_without_emails)
```

Comment:
Edge case, but are you ok that if there is username same as group name, group will take precedence?

Reply:
I think it is fine cause we try to share a namespace, you can not name a group the same as a user.